virtual_rainforest/models/hydrology/hydrology_model.py

```diff
@@ -448,6 +502,21 @@ def update(self, time_index: int) -> None:
             }
         )
 
+        # Calculate accumulated surface runoff
+        # Get the runoff created by SPLASH or initial data set
+        single_cell_runoff = np.array(soil_hydrology["surface_runoff"])
+
+        # Get accumulated runoff from previous time step
+        accumulated_runoff = np.array(self.data["surface_runoff_accumulated"])
+
+        # Calculate accumulated runoff for each cell (me + sum of upstream neighbours)
+        for cell_id, upstream_id in enumerate(self.upstream_ids):
+            accumulated_runoff[cell_id] += np.sum(single_cell_runoff[upstream_id])
```

Comment:
Given that you're repeating this code, it suggests it should live in its own function. I know it's only two lines of code, but it's not totally trivial and I think deserves a unit test.